doc/nrf/libraries/bluetooth/services/rrsp.rst

```diff
@@ -0,0 +1,59 @@
+.. _rrsp_readme:
+
+Ranging Responder (RRSP)
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+This library implements the Ranging Responder for Channel Sounding with the corresponding set of characteristics defined in the `Ranging Service Specification`_ and the `Ranging Profile Specification`_.
+
+Supported features
+==================
+
+- On Demand Ranging Data
+
+Configuration
+*************
+
+Enable RREQ and RRSP kconfig options with :kconfig:option:`CONFIG_BT_RAS` kconfig option.
+
+Set the maximum number of antenna paths supported by the device using the :kconfig:option:`CONFIG_BT_RAS_MAX_ANTENNA_PATHS` kconfig option.
+This sets the antenna paths per step that can be stored inside the Ranging Service.
+This value must match the supported Channel Sounding capabilities of the device.
+This affects the per-instance memory usage of the Ranging Service.
+
+Set support for storing Mode 3 Channel Sounding steps using the :kconfig:option:`CONFIG_BT_RAS_MODE_3_SUPPORTED` kconfig option.
+This will allocate memory for the Ranging Service to store Mode 3 Channel Sounding steps.
+This value must match the supported Channel Sounding capabilities of the device.
+This affects the per-instance memory usage of the Ranging Service.
+
+Enable RRSP kconfig options using the :kconfig:option:`CONFIG_BT_RAS_RRSP` kconfig option.
+
+Set new connections to be allocated a RRSP instance automatically using the :kconfig:option:`CONFIG_BT_RAS_RRSP_AUTO_ALLOC_INSTANCE` kconfig option.
+
+Set the number of simultaneously supported RRSP instances using the :kconfig:option:`CONFIG_BT_RAS_RRSP_MAX_ACTIVE_CONN` kconfig option.
+
+Set the number of ranging data buffers per connection using the :kconfig:option:`CONFIG_BT_RAS_RRSP_RD_BUFFERS_PER_CONN` kconfig option.
+
+Set the logging level of the RRSP library using the :kconfig:option:`CONFIG_BT_RAS_RRSP_LOG_LEVEL` Kconfig option.
```

Comment:
May be a bulleted list might be beneficial here in this section (in both libraries) :
Example: 
:kconfig:option:`CONFIG_BT_RAS_RRSP_RD_BUFFERS_PER_CONN`  - Sets the number of ranging data buffers per connection.
and so on.